tests/modules/test_file_batcher.py

```diff
@@ -54,8 +54,8 @@ def default_module_config_fixture():
         "module_name": "file_batcher",
         "namespace": MORPHEUS_MODULE_NAMESPACE,
         "sampling_rate_s": 0,
-        "start_time": "2022-08-01",
-        "end_time": "2022-08-31",
+        "start_time": "2022-08-01T00:00:00",
+        "end_time": "2022-08-31T00:00:00",
```

Comment:
Is this a breaking change?

Reply:
No, there are still other test date strings without a time component.